Packs/AzureDevOps/Integrations/AzureDevOps/AzureDevOps.yml

```diff
@@ -0,0 +1,1087 @@
+        the default value is 'unknown'.
+      type: String
+  - arguments:
+    - default: false
+      description: The name of the organizaion project.
+      isArray: false
+      name: project
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: The page number of the results to retrieve. Default is 1, minimum
+        is 1.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The number of results to retrieve. Default is 50, minimum is 1.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieve project pipelines list.
+    execution: false
+    name: azure-devops-pipeline-list
+    outputs:
+    - contextPath: AzureDevOps.Project.name
+      description: The name of the project.
+      type: String
+    - contextPath: AzureDevOps.Project.Pipeline.id
+      description: The ID of the pipeline.
+      type: Number
+    - contextPath: AzureDevOps.Project.Pipeline.revision
+      description: Pipeline revision number
+      type: Number
+    - contextPath: AzureDevOps.Project.Pipeline.name
+      description: Pipeline repository name
+      type: String
+    - contextPath: AzureDevOps.Project.Pipeline.folder
+      description: Pipeline folder
+      type: String
+  - arguments:
+    - default: false
+      description: The name of the organizaion project.
+      isArray: false
+      name: project
+      required: true
+      secret: false
+    - default: false
+      description: The name of the project repository.
+      isArray: false
+      name: repository
+      required: true
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: The page number of the results to retrieve. Default is 1, minimum
+        is 1.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The number of results to retrieve. Default is 50, minimum is 1.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieve repository branches list.
+    execution: false
+    name: azure-devops-branch-list
+    outputs:
+    - contextPath: AzureDevOps.Project.name
+      description: The name of the project.
+      type: String
+    - contextPath: AzureDevOps.Project.Repository.name
+      description: The name of the repository.
+      type: String
+    - contextPath: AzureDevOps.Project.Repository.Branch.name
+      description: The name of the branch
+      type: String
+    - contextPath: AzureDevOps.Project.Repository.Branch.creator
+      description: Thr creator of the branch.
+      type: String
+  dockerimage: demisto/crypto:1.0.0.24037
+  isremotesyncin: true
```

Comment:
In order to use_MicrosoftApiModule_

Reply:
Can you elaborate? we are not syncing in any events here.

Reply:
The docker image for Microsoft is demisto/crypto.
Because we use this APIModule, we have to configure the integration docker image to fit the API module's docker.
I checked this on several integrations that used the module - and they also used in demisto/crypto.
In addition, I think that as part of the CI-CD process, there is a requirement to use this docker image.

Reply:
No issue with the docker image, I am talking about the `isremotesyncin: true`